src/_cffi_src/openssl/asn1.py

```diff
@@ -113,6 +111,10 @@
 long ASN1_ENUMERATED_get(ASN1_ENUMERATED *);
 int i2a_ASN1_INTEGER(BIO *, ASN1_INTEGER *);
 
+/* This became const ASN1_TIME in 1.1.0f */
+ASN1_GENERALIZEDTIME *ASN1_TIME_to_generalizedtime(ASN1_TIME *,
```

Comment:
@levitte you'd think so but for various bad reasons that would cause other problems on older builds for us.